src/logging.ts

```diff
@@ -0,0 +1,52 @@
+import { EventEmitter } from "events";
+
+enum Level {
+  off,
+  error,
+  warn,
+  info,
+  debug,
```

Comment:
Vacillated between upper and lower case for these, in effect the only difference is the constructor/setter, and I thought lower case looked better. (`new BandwidthRtc("debug")` vs `new BandwidthRtc("DEBUG")`). This is a pretty basic/naive implementation, thoughts are welcome.